octobot_evaluators/evaluator/abstract_evaluator.py

```diff
@@ -160,13 +157,15 @@ def get_is_time_frame_wildcard(cls) -> bool:
         """
         pass
 
-    async def start_evaluator(self) -> None:
+    async def start_evaluator(self, in_backtesting:bool) -> None:
```

Comment:
What will be the use of `in_backtesting` param ?

Reply:
Mistake from me, it should be bot_id, I'm fixing it